### DISCUSSION TITLE
Last minute scheduling and the croncert run cycle

### DISCUSSION THREAD

**Discussion by MarkJaroski** [jakopako/croncert-config#208] [General]
I just noticed that Parenthese in Nyon added this on Tuesday :

```
{
  "city": "Nyon",
  "date": "2023-03-11T21:30:00+01:00",
  "location": "La Parenthèse",
  "sourceUrl": "http://www.bar-laparenthese.ch/",
  "title": "Bandit Voyage (CH) + Milla Pluton (CH) – Soirée Cheptel Records",
  "type": "concert",
  "url": "http://www.bar-laparenthese.ch/concerts/bandit-voyage-ch-milla-pluton-ch-soiree-cheptel-records/"
}
```

[Link](http://www.bar-laparenthese.ch/concerts/bandit-voyage-ch-milla-pluton-ch-soiree-cheptel-records/
)

I guess they probably want to keep the crowd size small, otherwise they would have announced it in advance, still, just FYI, Robin Giraud (Bandit Voyage) is one of the most interesting and versatile musicians working in Geneva, so it's a gig I don't want to miss.

**Comment by laerm** [comment]
Yeah, Robin Giraud is great, and very recognisable with his big hair :) First came across him in Mama Roisin, who were (are?) great too.

**Comment by MarkJaroski** [comment]
Were. The other guy, Cyril owns a record shop now (Bongo Joe).

**Comment by MarkJaroski** [comment]
I should mention that Robin's doing a lot of other stuff too, including a
free-jazz project *l'Orage *

https://www.locg.ch/fr/programme?saison_id=20&type_id=&concert_id=357

**Comment by jakopako** [comment]
Just watched a video of Bandit Voyage playing live and I liked it very much!

I'll kick off the github action that does the scraping manually so their concert is included on croncert.ch

Normally it runs every Saturday night. I could also increase the frequency to twice a week. I don't see any issues with that.
